tests/foreman/api/test_activationkey.py

```diff
@@ -424,9 +424,8 @@ def test_positive_remove_user():
     """
     password = gen_string('alpha')
     user = entities.User(password=password, login=gen_string('alpha'), admin=True).create()
-    cfg = get_nailgun_config()
-    cfg.auth = (user.login, password)
-    ak = entities.ActivationKey(cfg).create()
+    user_cfg = user_nailgun_config(user.login, password)
+    ak = entities.ActivationKey(server_config=user_cfg).create()
```

Comment:
We should evaluate if this is compatible with the satellite objects' entities.

Reply:
@JacobCallahan  Updated all files! Please revisit!